integration-test/test-pages.js

```diff
@@ -66,20 +53,22 @@ describe('Test integration pages', () => {
         )
         for (const key in pageResults) {
             for (const result of pageResults[key]) {
-                expect(result.result).withContext(key + ':\n ' + result.name).toEqual(result.expected)
+                expect(result.result).toEqual(result.expected)
```

Comment:
Can we not keep this context? It was useful for seeing the individual test failures.

Reply:
Nice:

```
await test.step(`${key}:\n ${result.name}`, async () => {
    expect(result.result).toEqual(result.expected);
});
```